.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Bump Version'
+
+on:
+  workflow_dispatch:
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.COMMIT_KEY }}
```

Comment:
I don't see that we have this secret configured :mag:. I guess we can remove this :eyes:...

```suggestion
```